src/Magento/FunctionalTestingFramework/Console/StaticChecksCommand.php

```diff
@@ -81,6 +88,7 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->ioStyle = new SymfonyStyle($input, $output);
+        $this->parseRulesetJson();
```

Comment:
ruleset should not be read if an argument is passed to the static-check command, correct?

Currently I see this:
```
Soumyas-MBP:magento2-functional-testing-framework sunnikri$ bin/mftf static-checks deprecatedEntityUsage
Using ruleset under /Users/sunnikri/workspace/magento2-functional-testing-framework/dev/staticRuleset.json

Running static check script for: deprecatedEntityUsage
```

Reply:
I see what you mean. In the code it will use arguments over the ruleset, but it's misleading messaging. I'll fix it.